### ISSUE TITLE
[a11y]: add required tab stop to multiline Codesnippet textbox

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#10527]
multiline codesnippet has an inaccessible scrollable text box. this text box should be able to receive focus and have a tab stop

**Comment by mbgower** [comment]
if it is implemented as a textbook, it should be read-only="true"

**Comment by mbgower** [comment]
Please notify me when this change is implemented, as we will need matching updates on the component's accessibility tab.

**Comment by jsehull** [comment]
@mbgower I know we chatted earlier, above is my submission for conversation if you want to share your thoughts.

**Comment by mbgower** [comment]
@jsehull is there a preview?

**Comment by jsehull** [comment]
@mbgower it does now! Hadn't loaded yet
